xarray_beam/_src/core_test.py

```diff
@@ -253,6 +253,32 @@ def test_dataset_to_chunks_multiple(self):
     )
     self.assertIdenticalChunks(actual, expected)
 
+  def test_datasets_to_chunks_multiple(self):
+    datasets = tuple(xarray.Dataset({'foo': ('x', np.arange(6))}) for _ in range(7))
```

Comment:
Here and below: can we make these datasets have data values, e.g.,
```suggestion
    datasets = tuple(xarray.Dataset({'foo': ('x', i + np.arange(6))}) for i in range(7))
```
This would result in better unit-tests.